File: Daily Challenge/Python/Difference - Same\Different Position 2.12.2025.py
n = int(input())
a = list(map(int, input().split()))

b = sorted(a)  # sorted copy

X = 0
Y = 0

for i in range(n):
    if a[i] == b[i]:
        X += a[i]
    else:
        Y += a[i]

print(abs(X - Y))

File: Daily Challenge/Java/Difference - Same\Different Position 2.12.2025.java
import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        
        int[] a = new int[n];
        for (int i = 0; i < n; i++) a[i] = sc.nextInt();
        
        int[] b = a.clone();      // copy original
        Arrays.sort(b);           // sorted array
        
        int X = 0, Y = 0;
        
        for (int i = 0; i < n; i++) {
            if (a[i] == b[i]) 
                X += a[i];        // same position
            else 
                Y += a[i];        // changed position
        }
        
        System.out.println(Math.abs(X - Y));
    }
}